README.md

```diff
@@ -13,72 +13,101 @@ To get business, simply:
 $ gem install business
 ```
 
+## Important: 2.0.0 breaking changes
+
+We have removed the bundled calendars as of version 2.0.0, if you need the calendars that were included:
+
+Download the calendars you wish to use from [before version 2](https://github.com/gocardless/business/tree/b12c186ca6fd4ffdac85175742ff7e4d0a705ef4/lib/business/data) and place them in a suitable place in your project.
```

Comment:
Are we able to provide a perm link to the old data? Concerned that the sha in the link may shift and result in a 404.

Reply:
The SHA Is how github generates permanent links. One of the guarantees Git gives you is that the SHA will never shift and Github just exposes that.

Reply:
This is the permalink, press `y` on any file you're viewing to expand the link to a permalink